src/main/java/com/patchworkmc/Constants.java

```diff
@@ -0,0 +1,31 @@
+package com.patchworkmc;
+
+import org.objectweb.asm.Handle;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.Type;
+
+public final class Constants {
```

Comment:
I'm not a huge fan of this class, it feels like it combines somewhat unrelated things together.

* `EventBus` should probably be a separate class within the event package.
* `Lambdas` would probably be better as part of a class with utility methods for creating invokedynamic instructions.